unlock-js/src/__tests__/web3Service.test.js

```diff
@@ -658,13 +515,80 @@ describe('Web3Service', () => {
       return nock.resolveWhenAllNocksUsed()
     }
 
+    describe('inputsHandlers', () => {
```

Comment:
these tests don't use the contract at all, they should live outside the `describe.each`